src/components/Detail/Detail.js

```diff
@@ -28,6 +29,9 @@ const Detail = () => {
           <span>price</span>
           <span>{room.price}</span>
         </div>
+        <div>
+          <DeleteRoom name="Delete room" id={room.id} />
+        </div>
```

Comment:
Kindly change the delete room button for a reserve button and add its functionality